llm_benchmark/benchmark/vllm_benchmark/benchmark_serving.py

```diff
@@ -468,9 +483,11 @@ async def benchmark(
         "total_output_tokens": metrics.total_output,
         "request_throughput": metrics.request_throughput,
         "output_throughput": metrics.output_throughput,
+        "otpr":[output.output_throughput_user for output in outputs],
         "total_token_throughput": metrics.total_token_throughput,
         "input_lens": [output.prompt_len for output in outputs],
         "output_lens": actual_output_lens,
+        "e2els":[ output.latency for output in outputs],
```

Comment:
cross check the naming convention against the main branch, if not please change this to match that of main branch